packages/env-utils/src/__tests__/envUtils.test.ts

```diff
@@ -1,6 +1,5 @@
-import { isAndroid, isChromeOs } from '@trezor/env-utils';
-import { isWindows, isMacOs, isLinux, getOsName, isIOs } from '@suite-utils/env';
-import * as fixtures from '../__fixtures__/env';
+import { isAndroid, isChromeOs, isWindows, isMacOs, isLinux, getOsName, isIOs } from '../envUtils';
```

Comment:
I would suggest removing this tests completely. IMHO they have no value because env utils functions are usually just one `if` and mocking browser functions here is like testing `expect(true).toBeTruthy()`

Reply:
Would be enough IMHO

Reply:
Looks like it is not required anymore.

Reply:
removed, rebased and squashed